src/index.tsx

```diff
@@ -87,6 +88,10 @@ class CirculationAdmin {
                 path="/admin/web/dashboard(/:library)"
                 component={DashboardPage}
               />
+              <Route
+                path="/admin/web/quicksight(/:library)"
```

Comment:
Not sure why we're passing the library here, since it is not currently used to select which `libraryUuid`s are selected for inclusion.

Reply:
Right - I'll remove that for now.   Oversight.